docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -6,95 +6,95 @@ All gateway events in Discord are tagged with an opcode that denotes the payload
+| 4015 | Voice server crashed    | The server crashed. Our bad! Try [resuming](#DOCS_RESOURCES_VOICE_CONNECTIONS/resuming-voice-connection).                                           |
+| 4016 | Unknown encryption mode | We didn't recognize your [encryption](#DOCS_RESOURCES_VOICE_CONNECTIONS/encrypting-and-sending-voice).                                              |
 
 ## HTTP
 
 Our API will return semantically valid HTTP response codes based on the success of your request. The following table can be used as a reference for response codes it will return.
 
 ###### HTTP Response Codes
 
-| Code                      | Meaning                                                                                |
-| ------------------------- | -------------------------------------------------------------------------------------- |
-| 200 (OK)                  | The request completed successfully.                                                    |
-| 201 (CREATED)             | The entity was created successfully.                                                   |
-| 204 (NO CONTENT)          | The request completed successfully but returned no content.                            |
-| 304 (NOT MODIFIED)        | The entity was not modified (no action was taken).                                     |
-| 400 (BAD REQUEST)         | The request was improperly formatted, or the server couldn't understand it.            |
-| 401 (UNAUTHORIZED)        | The `Authorization` header was missing or invalid.                                     |
-| 403 (FORBIDDEN)           | The `Authorization` token you passed did not have permission to the resource.          |
-| 404 (NOT FOUND)           | The resource at the location specified doesn't exist.                                  |
-| 405 (METHOD NOT ALLOWED)  | The HTTP method used is not valid for the location specified.                          |
-| 429 (TOO MANY REQUESTS)   | You are being rate limited, see [Rate Limits](#DOCS_TOPICS_RATE_LIMITS/rate-limits).   |
-| 502 (GATEWAY UNAVAILABLE) | There was not a gateway available to process your request. Wait a bit and retry.       |
-| 5xx (SERVER ERROR)        | The server had an error processing your request (these are rare).                      |
+| Code                      | Meaning                                                                              |
+|---------------------------|--------------------------------------------------------------------------------------|
+| 200 (OK)                  | The request completed successfully.                                                  |
+| 201 (CREATED)             | The entity was created successfully.                                                 |
+| 204 (NO CONTENT)          | The request completed successfully but returned no content.                          |
+| 304 (NOT MODIFIED)        | The entity was not modified (no action was taken).                                   |
```

Comment:
What cases do you return this? Usually you return a 204 if nothing has changed, right?

Reply:
I can look into this